src/types.ts

```diff
@@ -326,6 +331,13 @@ export type IMailboxSetArguments = ISetArguments<IMailboxProperties>;
 
 export type IMailboxSetResponse = ISetResponse<IMailboxProperties>;
 
+/**
+ * See https://jmap.io/spec-core.html#method-level-errors
+ */
+export interface IError {
+  type: IErrorType;
```

Comment:
Is this some default typing provided by TypeScript? If not, then I haven't found it in this PR.

Reply:
Was already present before the PR https://github.com/OpenPaaS-Suite/jmap-client-ts/blob/main/src/types.ts#L332